ts/features/fci/store/reducers/fciSignatureDetailView.ts

```diff
@@ -0,0 +1,37 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
```

Comment:
Isn't better to replace all the "signatureDetail" to "signatureRequest" that is the real info you are storing?
Apply also for the file name.